app/controllers/application_controller.rb

```diff
@@ -3,13 +3,25 @@
 class ApplicationController < ActionController::Base
   before_action :decode_json_api
 
-  rescue_from ActiveRecord::RecordNotFound do |_exception|
-    render_error(ApiError.new(:id, 'Not found.'), :not_found)
+  rescue_from ActiveRecord::RecordNotFound do |exception|
+    render_api_error(exception.to_s, :not_found)
```

Comment:
Is there a reason for using `exception.to_s` here and `exception.message` in the other cases below? I'm assuming they do the same thing, in which case consistency is nice.